crates/uv/src/commands/pip_install.rs

```diff
@@ -192,6 +192,7 @@ pub(crate) async fn pip_install(
         .connectivity(connectivity)
         .index_urls(index_locations.index_urls())
         .keyring_provider(keyring_provider)
+        .markers(markers)
```

Comment:
Thanks for the edits @konstin

I think this is missing adding `.platform(venv.interpreter().platform())`

Reply:
Thank you! #2507